Packs/Infoblox/Integrations/Infoblox/Infoblox.py

```diff
@@ -312,6 +422,37 @@ def delete_rpz_rule(self, reference_id: Optional[str]) -> Dict:
         suffix = reference_id
         return self._http_request('DELETE', suffix)
 
+    def get_host_records(self, name: str | None) -> dict:
+        """
+        Get the host records.
+
+        Args:
+        - `name` (``str``): Name of the host record to search for.
+
+        Returns:
+        - Response JSON
+        """
+
+        params = assign_params(name=name)
+        return self._http_request('GET', self.GET_HOST_RECORDS_ENDPOINT, params=params)
+
+    def get_network_info(self, pattern: str | None) -> dict:
+        """
+        Get the network information.
+
+        Args:
+        - `host_name` (``str | None``): The hostname to retrieve network information for.
+        - `pattern` (``str | None``): Filter networks by pattern, e.g. '.0/24' for netmask, '192.168' for subnet.
+
+        Returns:
+        - Response JSON
+        """
+
+        if pattern:
+            return self._http_request("GET", self.NETWORK_ENDPOINT, params={"network~": pattern})
+        else:
+            return self._http_request("GET", self.NETWORK_ENDPOINT)
+
```

Comment:
Not sure I understand why you need this if else. what is the difference between the above to just this:
```suggestion
        params={"network~": pattern} if pattern else None
        return self._http_request("GET", self.NETWORK_ENDPOINT, params=params)
```